orttraining/orttraining/models/runner/training_runner.cc

```diff
@@ -679,6 +774,10 @@ Status TrainingRunner::UpdateParams(Parameters params) {
   return Status::OK();
 }
 
+void TrainingRunner::UpdateMetricsParams(MapStringToString& properties) {
```

Comment:
could be `const MapStringToString& properties`.

also, how about just passing in this perf properties map to TrainingRunner::Run()? could have an empty default value. then we don't need to keep it in TrainingParameters and need to call a separate function to set it

Reply:
will check and see ... thx